xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -366,9 +528,19 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value that indicates which value property <c>Value</c> property to use to get the token value.</summary>
```

Comment:
How would this show up in the docs? Looks like `value property` is repeated twice.

Reply:
I should have removed that. But you can see how the API docs are shaping up at https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json?view=netcore-3.0&branch=pr-en-us-1880

Doing the code style blocks that word from loc - that was my intention at least.

![image](https://user-images.githubusercontent.com/12971179/53224154-9fda8200-3628-11e9-95a0-f15a8fd3bd46.png)

Reply:
Thanks for the link.

Are all ref structs listed as regular structs? It looks like it (same issue with Span). Is there a way for us to fix that?

![image](https://user-images.githubusercontent.com/6527137/53224824-e8933a80-362a-11e9-82c2-4982d164eed6.png)

Reply:
It's a known issue with our reflection tool: https://github.com/mono/api-doc-tools/issues/260

@joelmartinez and @TianqiZhang might know better when that should be coming

Reply:
It is on the roadmap, though I don't have a specific ETA for it. /cc @wh-alice